cpp/tests/io/json_test.cpp

```diff
@@ -915,13 +915,30 @@ TEST_F(JsonReaderTest, BadDtypeParams)
   EXPECT_THROW(cudf_io::read_json(options_map), cudf::logic_error);
 }
 
-TEST_F(JsonReaderTest, ExperimentalParam)
+TEST_F(JsonReaderTest, JsonExperimentalBasic)
 {
-  cudf_io::json_reader_options const options =
-    cudf_io::json_reader_options::builder(cudf_io::source_info{nullptr, 0}).experimental(true);
+  const std::string fname = temp_env->get_temp_dir() + "JsonLinesFileTest.json";
+  std::ofstream outfile(fname, std::ofstream::out);
+  outfile << "[{\"a\":\"11\", \"b\":\"1.1\"},{\"a\":\"22\", \"b\":\"2.2\"}]";
```

Comment:
Might look slightly cleaner using a raw string literal:
```suggestion
  outfile << R"([{"a":"11", "b":"1.1"},{"a":"22", "b":"2.2"}])"
```